Application/mobile/app_code/lib/l10n/app_fr.arb

```diff
@@ -75,12 +76,15 @@
   "errorOccurredDuringUpdateUserInformation": "Une erreur est survenue lors de la mise à jour des informations de l'utilisateur.",
   "fieldRequired": "Ce champ est requis.",
   "fieldsEmpty": "Certains champs sont vides.",
+  "fillComment": "Veuillez ajouter un commentaire.",
   "firstName": "Prénom",
+  "french": "Français",
```

Comment:
I preshot the review of Tanguy, even in french english text should be "English" instead of "Anglais"